pyproject.toml

```diff
@@ -0,0 +1,60 @@
+[tool.poetry]
+name = "semversioner"
+version = "2.0.2"
```

Comment:
Hmm, that may be an incompatible goal with how Poetry works

Reply:
Hmm, I agree that there sholud be a single source of truth here, but I kind of think it should go the other way.
 
We could model this as a build step that reads from pyproject.toml and injects that version into __init__.py.

Another route we could go is use the `poetry-dynamic-versioning` plugin to hydrate the version in pyproject.toml and in __version__.py the closest git tag, which is what I used recently in this project (https://github.com/graspologic-org/graspologic).

Reply:
Yes, makes sense. We could update both places during the build process: https://github.com/raulgomis/semversioner/blob/master/.github/workflows/build-and-publish.yml#L60 and have an additional check to verify both versions are the same before publishing to avoid any mismatch.

This is an interesting thread with similar (and additional) ideas: https://github.com/python-poetry/poetry/issues/144#issuecomment-485287982 .

Thoughts?